documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,83 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Advantages and disadvantages of cold and warm migrations
+
+[id="warm-cold-migration-overview_{context}"]
+=== Overview
+
+Both cold migration and warm migration have advantages and disadvantages, as described in the table that follows:
+
+[cols="1,1,1",options="header"]
+.Advantages and disadvantages of cold and warm migrations
+
+|===
+| |Duration |VM downtime
```

Comment:
We need to update this table

Method | Duration | Data transferred | VM Downtime
--|--|--|--
Cold migration | Correlates to amount of disk data | Approx sum of all disks | High
Warm migration | Correlates to amount of disk data and VM utilization | Approx sum of all disks and changed blocks | Low

Reply:
@fabiand  I added the Data transferred rows and changed the contents of the rows as you suggested; I did not remove any rows.

Reply:
@mnecas Please check now. I forgot to push my changes.